### PR TITLE
Integrate datasets to experiment_api

### PR DESCRIPTION
**Pull request by tiffapedia**
Integrate datasets to experiment_api.
Create Veri dataset.
Add additional tests as necessary.

### CODE REVIEW

**Comment by agude** [comment]
Can we break this into two PRs? One for VeRI, and one for the Experiment API? It would make it easier to look through!

File: pelops/experiment_api/experiment.py [hunk 1]
```diff
@@ -51,9 +53,11 @@
     -s, --cams=<NUM_CAMS>           : Each camera maps to a set. NUM_CAMS specify the number of camera sets to be outputted.
     -c, --cars=<NUM_CARS_PER_CAM>   : Each set has a list of images. NUM_CARS_PER_CAM specify the number of car images in each camera set.
     -d, --drop=<DROP_PERCENTAGE>    : The likelihood that the target car image is dropped (float from [0,1])
-    -y, --type=<TYPE>               : Determine which type of images to use.
-                                      0: all, 1: query, 2: test, 3: train
     -e, --seed=<SEED>               : Seed to be used for random number generator.
+    -y, --stype=<TYPE>              : Determine which type of images to use.
+                                      0: all, 1: query, 2: test, 3: train
+    -w, --dtype=<DATASET_TYPE>      : Specify the datasets to use.
```
Comment (agude) [inline pelops/experiment_api/experiment.py]: No need to abreviate in the long flag, you can do `--dataset-type` and `--set-type` or similar.

Comment (tiffapedia) [inline pelops/experiment_api/experiment.py]: Those are old comments for docopt. They are not exactly how argparse handles it. Argparse just takes in -y or -w. If you preferred the longer flag, let me know.

**Comment by tiffapedia** [comment]
We cannot break the PR by commits because the changes made for data integration effects the Veri dataset creation. I can cherry-pick the commits, but they're going to make history changes in the git log. If it helped, the only thing for Veri dataset creation is in pelops/pelops/datasets/veri.py

File: pelops/experiment_api/experiment.py (hunk 1 above)
Comment (tiffapedia) [inline pelops/experiment_api/experiment.py]: I have recently committed to reflect what argparse takes in

**Pull request merged**